GenderPayGap.WebUI/Views/Submit/EmployerWebsite.cshtml

```diff
@@ -61,9 +61,10 @@
                 <p>You can publish your gender pay gap narrative on your organisation's website. This can provide context for your figures.</p>
                 <fieldset>
                     <div class="form-group @Html.SetErrorClass(model => model.CompanyLinkToGPGInfo, "error")">
-                        @Html.LabelFor(model => model.CompanyLinkToGPGInfo, "Enter the web address (URL)", new {@class = "form-label"})
+                        <label class="form-label" for="OtherName">Enter the web address (URL). <br/>
+                            <span class="optional" style="font-size: inherit">This must start with http:// or https://</span></label>
                         @Html.ValidationMessageFor(model => model.CompanyLinkToGPGInfo, "", new {@class = "error-message"})
-                        @Html.TextAreaFor(model => model.CompanyLinkToGPGInfo, 6, 30, new {@class = "form-control form-control-3-4"})
+                        @Html.TextAreaFor(model => model.CompanyLinkToGPGInfo, 6, 30, new {@class = "form-control form-control-3-4", placeholder = "http"})

```

Comment:
This page is still used by the old submission controller so I didn't want to update to use new components. I'm aware that this is not the best solution, not least for accessibility. 

I thought it was the best use of time for now, but I'm happy to find a different approach.